### PR TITLE
refactor(nav-drawer): schema colors

### PR DESCRIPTION
**Pull request by desig9stein**

### CODE REVIEW

File: sass/themes/schemas/components/dark/_navdrawer.scss [hunk 1]
```diff
@@ -53,9 +53,8 @@ $dark-fluent-navdrawer: extend(
         ),
         item-active-background: (
             color: (
-                'primary',
+                'gray',
                 300,
-                0.12,
```
Comment (didimmova) [inline sass/themes/schemas/components/dark/_navdrawer.scss:58]: it still has 0.12 opacity

**Comment by simeonoff** [comment]
@desig9stein wait, why is this PR targeting the indigo branch?

File: sass/themes/schemas/components/dark/_navdrawer.scss (hunk 1 above)
Comment (desig9stein) [reply sass/themes/schemas/components/dark/_navdrawer.scss:58]: item active background is 300 in the UI kit.

Screenshot 2024-04-23 at 20 41 13

**Comment by desig9stein** [comment]
> @desig9stein wait, why is this PR targeting the indigo branch?

@simeonoff I made a mistake, if I change the base it gets messy. Do you mind if I create a new PR from the master branch and cherry-pick all my commits?

File: sass/themes/schemas/components/dark/_navdrawer.scss (hunk 1 above)
Comment (didimmova) [reply sass/themes/schemas/components/dark/_navdrawer.scss:58]: No, it's not. Ask somebody from the design team if you're not sure.

Screenshot 2024-04-24 at 8 14 38

Comment (desig9stein) [reply sass/themes/schemas/components/dark/_navdrawer.scss:58]: My bad, I didn't inspect the correct element.

**Comment by desig9stein** [comment]
@simeonoff @didimmova i created a new branch [here](https://github.com/IgniteUI/igniteui-theming/pull/252)

**Pull request closed without merging**